client/db.go

```diff
@@ -374,8 +376,14 @@ func (db *DB) Run(b *Batch) error {
 //
 // TODO(pmattis): Allow transaction options to be specified.
 func (db *DB) Tx(retryable func(tx *Tx) error) error {
+	_, file, line, ok := runtime.Caller(1)
```

Comment:
I'll benchmark. Pretty sure it is orders of magnitude less expensive than an RPC.